tests/testthat/test-fit_percentiles.R

```diff
@@ -0,0 +1,39 @@
+test_that("Test if checkmate checks work", {
+  skip_if_not_installed("withr")
+  withr::local_seed(42)
+  obs <- 10
+  season <- c("2018/2019", "2019/2020", "2020/2021")
+  season_num_rev <- rev(seq(from = 1, to = length(season)))
```

Comment:
FYI: This should also work

The `seq_len()` and `seq_along()` functions are very useful IMO
```suggestion
  season_num_rev <- rev(seq_along(season))
```